app/assets/javascripts/user-satisfaction-survey.js

```diff
@@ -67,12 +67,21 @@
     setSurveyUrl: function(href) {
       var $surveyLink = $('#take-survey');
       var surveyUrl = $('#user-satisfaction-survey-container').data('survey-url');
+      var surveyStarts = new Date(2016, 1, 26).getTime();
+      var surveyEnds = new Date(2016, 1, 27, 23, 59, 59).getTime();
+
+      if (userSatisfaction.currentDate() >= surveyStarts &&
+          userSatisfaction.currentDate() <= surveyEnds) {
+        surveyUrl = 'https://www.surveymonkey.co.uk/r/2MRDLTW';
+      }
```

Comment:
@benlovell chatted to @fofr and I about this. We were both happy with one URL being in the JS, given this is a very short term thing.

We're generally moving towards [the pattern of JS being logic-only, and pulling configuration from markup](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#use-data-attributes-for-configuration), so if this were a longer-lived implementation, I'd suggest putting both URLs in markup.